test/output/test-file.json

```diff
@@ -0,0 +1,5 @@
+[
```

Comment:
If the comment above about possibly using `test-data.json` rather than `test-file.json` this file can be removed I think?